tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -97,6 +103,19 @@ def add_row_item(run, tag=None):
             'profile': False})
       return (run_item, tag_item)
 
+    if self._data_provider:
+      mapping = self._data_provider.list_blob_sequences(
+          experiment_id=experiment,
```

Comment:
Aha: this was seemingly working here because `MultiplexerDataProvider` ignores `experiment_id`, and so is perfectly happy receiving `None`.  So I think it is not strictly true that you can't ever pass a `None` experiment to a data provider.  This is all a bit entangled/contingent; I think it could be more cleanly factored, but not now (maybe we can discuss when we have bandwidth for tech debt). `ScalarsPlugin.index_impl()` has the same issue.

Anyway, for now, yes I'll just make `is_active()` always return true in the data provider case.